tests/snmpsim/akcp-securityprobe.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|securityProbe
```

Comment:
This needs to be the full sysDescr. I can see some examples in our stats:

securityProbe SEC-MX25V405n Jun 02 2016 10:57:49
securityProbe 5E SEC-PXAV405j Jun 25 2015 12:45:48